hnix.cabal

```diff
@@ -940,31 +940,31 @@ executable hnix
       main
   ghc-options: -Wall -rtsopts
   build-depends:
-      aeson
-    , base
-    , base16-bytestring
-    , bytestring
-    , comonad
-    , containers
-    , data-fix
-    , deepseq
-    , exceptions
-    , filepath
-    , free
+      hnix
+    , aeson >= 1.4.2 && < 1.6
```

Comment:
You don't have to duplicate bounds that already exist elsewhere in the cabal file.  You only need to put bounds for the dependencies here that don't have bounds elsewhere.

Reply:
You are right.

I would redo.

Reply:
Thanks, indeed - bounds were normal, additionally made bounds unique.